test/powershell/Modules/Microsoft.PowerShell.Utility/Select-String.Tests.ps1

```diff
@@ -4,245 +4,246 @@
 Describe "Select-String" -Tags "CI" {
     $nl = [Environment]::NewLine
```

Comment:
line 5 and 6 should really be in a `BeforeAll{}`

Reply:
added